swift_scality_backend/diskfile.py

```diff
@@ -311,9 +317,8 @@ def merge_df_mf_metadata(df_md_source, mf_md_source):
             md_dest.update(df_md_source)
         else:
             sys_metadata = {
-                key: val for key, val in df_md_source.items()
-                if key.lower() in (RESERVED_DATAFILE_META | DATAFILE_SYSTEM_META) or
-                is_sys_meta('object', key)
+                key: val for key, val in df_md_source.items() if key.lower() in
```

Comment:
What does this bring? I think the original (with newlines) is more readable...

Reply:
:+1: